requirements/dev.txt

```diff
@@ -183,8 +178,15 @@ matplotlib-inline==0.1.6
     #   kytos
 mccabe==0.6.1
     # via pylint
-packaging==20.3
+mongomock==4.1.2
```

Comment:
@Ktmi, after looking into [`mongomock`](https://github.com/mongomock/mongomock) looks like it's not actively being maintained, PRs and issues stalled since August or with non existent reviews or feedback on issues:

![20221128_112853](https://user-images.githubusercontent.com/1010796/204303539-ce1950ba-c6f5-41e2-95b6-032e26826b59.png)

![20221128_112919](https://user-images.githubusercontent.com/1010796/204303867-2a3fd04b-977d-43a8-8088-3e9b0e92542e.png)

![20221128_113113](https://user-images.githubusercontent.com/1010796/204304093-29af93c2-89ae-4c23-8f52-ec7ecfc25123.png)

With that said, this can turn into vulnerabilities or expensive for our team to maintain later on even though it's only a development dependency. The direction of mockmongo the project seems unknown to me, so when we upgrade to newer versions who knows how this will behave. I think it's risky to include this all things considered, I'll suggest to keep using the existing MagicMock on NApps, although not ideal, they aren't that bad either I think, especially with e2e tests that we also have full database coverage. 

Let me know what you think or if you have other considerations. Maybe we can keep a new issue/enhancement to try to keep open to potential other alternatives if any and if it's worth it.

Reply:
You make a good point. However, I don't think database operations are something we can do normal mock testing with. Asking a mock if db statement was called with a particular set of parameters, doesn't provide the same insight as checking the db for the appropriate side effect. Checking for the right side effects needs either a live server, or a specialized mock of the client in order to test. Since mongomock appears to have weak support, we still have the alternative of a live test server.

Reply:
> same insight as checking the db for the appropriate side effect

@Ktmi that's true for all mocked interfaces. If we had a really first class official well supported mocking library, like same level as AWS provides for their services `boto3` -> `moto` , I believe that'd be worth it, otherwise, as a core dependency even for testing is risky to maintain/introduce this considering the benefits. Unfortunately, I won't be able to accept this PR. 

The idea is great, and we can keep an issue to try to enhance this in a next opportunity, or if this library in the future starts to be more actively maintained and if the majority of the team feel comfortable with htis, otherwise, introducing a core library we're pretty much recommending/encouraging it, and for the core and NApps, I don't feel confident with this dependency in its current state. 

Thanks, David.